x-pack/plugins/security_solution/server/lib/detection_engine/routes/fleet/get_installed_integrations/get_installed_integrations_route.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformError } from '@kbn/securitysolution-es-utils';
+
+import type { SecuritySolutionPluginRouter } from '../../../../../types';
+import { DETECTION_ENGINE_INSTALLED_INTEGRATIONS_URL } from '../../../../../../common/constants';
+import { GetInstalledIntegrationsResponse } from '../../../../../../common/detection_engine/schemas/response/get_installed_integrations_response_schema';
+import { buildSiemResponse } from '../../utils';
+import { createInstalledIntegrationSet } from './installed_integration_set';
+
+/**
+ * Returns an array of installed Fleet integrations and their packages.
+ */
```

Comment:
May be worth adding a quick summary on the current method for collecting this information -- that we're first querying all `packagePolicies` to determine all installed integrations, then querying for each applicable package to augment with additional data.